### PR TITLE
Adds SilencedMP5A5's fluff items

### PR DESCRIPTION
**Pull request by Cameron653**
https://forum.vore-station.net/viewtopic.php?f=27&t=1302

### CODE REVIEW

**Comment by Cameron653** [comment]
Correction: Need to update the name to be NT-HASD instead of serdykov antoz.

**Comment by Cameron653** [comment]
Fixed.

**Comment by SilencedMP5A5** [comment]
err, the way the suit is currently means anyone can put it on
like, even nontaurs
is there a way to fix this?

**Comment by KasparoVy** [comment]
What do you mean? It checks for ckey.
https://github.com/VOREStation/VOREStation/pull/3406/files#diff-746910cb10d59a2f75a8b280085ab23eR1777

**Comment by Cameron653** [comment]
Hmm, I can add a check to make sure the user is a taur like the normal taur hardsuits.

**Comment by Cameron653** [comment]
Done. Added a check to see if they're a synthlizard & that their ckey is yours.

File: code/modules/clothing/spacesuits/void/void_vr.dm [hunk 1]
```diff
@@ -368,3 +368,21 @@
 			else
 				H << "<span class='warning'>You need to have a horse, wolf, or naga half to wear this.</span>"
 				return 0
+
+/obj/item/clothing/suit/space/void/security/hasd
```
Comment (Arokha) [inline code/modules/clothing/spacesuits/void/void_vr.dm]: Why is this not in custom_clothes_vr.dm? It's ckey locked, so it definitely should be.

Comment (Cameron653) [reply code/modules/clothing/spacesuits/void/void_vr.dm]: Ack, let me move that. Forgot about that.

File: code/modules/clothing/spacesuits/void/void_vr.dm [hunk 2]
```diff
@@ -134,7 +134,7 @@
 			else
 				H << "<span class='warning'>You need to have a horse, wolf, or naga half to wear this.</span>"
 				return 0
-				
+
```
Comment (Cameron653) [inline code/modules/clothing/spacesuits/void/void_vr.dm]: I don't know why these lines were touched. They don't do anything, though.

**Comment by Arokha** [comment]
Fixy da file

**Comment by Cameron653** [comment]
File fixed

**Pull request merged**